app/models/concerns/smooch_messages.rb

```diff
@@ -568,5 +568,10 @@ def min_number_of_words_for_tipline_long_text
       # Define a min number of words to create a media
       CheckConfig.get('min_number_of_words_for_tipline_long_text') || CheckConfig.get('min_number_of_words_for_tipline_submit_shortcut', 10, :integer)
     end
+
+    def replace_placeholders(uid, text)
+      external_id = TiplineMessage.where(uid: uid).last&.external_id
```

Comment:
Just to be super safe, I suggest you filter by `team_id` as well.

Reply:
Done